content/docs/start/data-and-model-access.md

```diff
@@ -1,13 +1,16 @@
 ---
-title: 'Get Started: Data Access'
+title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data Access
+# Get Started: Data and Model Access
 
-Okay, now that we've learned how to _track_ data and models with DVC and how to
-version them with Git, next question is how can we _use_ these artifacts outside
-of the project? How do I download a model to deploy it? How do I download a
-specific version of a model? How do I reuse datasets across different projects?
+Okay, now that we've learned how to _track_ data files in DVC and how to version
+them with Git. _Models_ in a machine learning project are also files written and
```

Comment:
> Okay, now that we've ... with Git.

Incomplete sentence

Reply:
Actually _version_ is a verb there. Fixed in 09dc8ca7c498ee0414c3b6bce7430cc82054f012. Thanks.

Reply:
Still incomplete. "to version" was OK. The part that is messing the sentence up is "now that". Please either remove that or complete the sentence after "in Git, (something)" (or rewrite in some other complete way). Thanks

Reply:
Removed `Okay, now that` in b46cca31